packages/core/src/components/search/search.scss

```diff
@@ -0,0 +1,31 @@
+@use '../../helpers';
+
+@mixin Search() {
+  .ods-search {
+    align-items: center;
+    display: inline-flex;
+    position: relative;
+    width: helpers.space(50);
+
+    .ods-input {
+      border-radius: helpers.border-radius('full');
+      padding-left: helpers.space(2) + helpers.space(0.5);
```

Comment:
I assume it's temporary, we'll wait for Steve's input.
In order to align text inside the Input with the full border-radius, 20px is needed for the left padding 🤷🏻 
If this is the value we end up going with, we should probably allow `space(2.5)`.

Reply:
Right, well I assume the improved design will not have such border radius - looks way off other designs like input.

Reply:
Yea probably so, I'll leave this conversation open so we don't forget to remove this code atrocity 😂

Reply:
I've allowed `2.5` in `space` as it seems that's the correct design. Thoughts?

Reply:
I don't like it... Seems like loosing purpose of 8px grid.

Maybe such cases better be hardcoded onto component, otherwise such now allos to use spacing in all apps.

Reply:
To be fair I'm already using 20px in my team's product, so I'd benefit from it being allowed.

Another thing we didn't consider is negative values, like `space(-0.5)` isn't allowed but it probably should be.

I don't mind hard coding the value here, but it feels like we need to review `space`.

Reply:
On the top of my head, probably `space` should not restrict on what multiplier is being passed? It should (probably) only do the calculation of base, then it's up to whoever is using to to follow the standard or not...

Reply:
Yea I feel that way too, I don't like runtime checks anyway.
How do you feel about this thread @Subcide?

Reply:
Let's move this to a discussion at later stage - resolving this now.